src/main/webapp/app/iris/settings/iris-settings-update/iris-common-sub-settings-update/iris-common-sub-settings-update.component.ts

```
@@ -101,12 +114,12 @@
         this.onEnabledChange();
     }
 
-    onInheritAllowedModelsChange() {
-        if (this.inheritAllowedModels) {
-            this.subSettings!.allowedModels = undefined;
-            this.allowedIrisModels = this.getAvailableModels();
+    onInheritAllowedVariantsChange() {
+        if (this.inheritAllowedVariants) {
+            this.subSettings!.allowedVariants = undefined;
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)